faker/providers/internet/__init__.py

```diff
@@ -5,10 +5,11 @@
 
 from .. import BaseProvider
 
-from ipaddress import ip_address, ip_network, IPV4LENGTH, IPV6LENGTH
+from ipaddress import ip_address, ip_network, IPV4LENGTH, IPV6LENGTH, _IPv4Constants, _IPv6Constants
```

Comment:
I don't htink we should use undocumented feature of the stdlib. They might change in the future and break `Faker`